lib/bunny/exceptions.rb

```diff
@@ -97,6 +97,12 @@ def initialize(frame)
     end
   end
 
+  class ConnectionManuallyClosed < Exception
```

Comment:
Other clients use `AlreadyClosedException` (Java) and similar names. I'd prefer that.